### ISSUE TITLE
Configurable for multispectral images?

### ISSUE THREAD

**Issue by JohnMrziglod** [Algy/fast-slic#10]
Thank you for this package. This is awesome.

I was wondering - since I work with multispectral imagery (more than three channels) - whether it is possible to simply adapt your package for this use case?

Cheers

**Comment by everettvacek** [comment]
I agree! Making it work with channels > 3 and double precision would be super helpful.
